extensions/cornerstone/src/types/Presentation.ts

```diff
@@ -1,23 +1,23 @@
 /** Store presentation data for either stack viewports or volume viewports */
 import { Types } from '@cornerstonejs/core';
+import { Types as UITypes } from '@ohif/ui';
 
-export interface BasePresentation {
-  id: string;
-  properties: Record<string, unknown>;
-  initialImageIndex?: number;
+/**
+ * Has information on the presentation of the viewport.
+ */
+export interface Presentation extends Types.StackViewportProperties {
+  presentationIds: UITypes.PresentationIds;
+  viewportType: string;
+  initialImageIndex: number;
   camera: Types.ICamera;
+  properties: Types.StackViewportProperties | Types.VolumeViewportProperties;
+  zoom?: number;
```

Comment:
Are not zoom and pan already "in" the camera?

Reply:
Yes, but they apply differently, and are something that can be applied to a different display set, so it is worth preserving them to allow extensions to apply them later.